lib/react_on_rails/server_rendering_pool.rb

```diff
@@ -106,6 +106,12 @@ def console_polyfill
 });
         JS
       end
+
+      def react_on_rails
+        return ::Rails.application.assets["react_on_rails.js"] unless Rails.env.production?
```

Comment:
Maybe we should just check the return value, and if not sufficient, then use the next statement? I'm concerned that we'll break those using older versions of Sprockets. Additionally, production environments could have other names than "production". I've often deployed apps with "staging" as the environment.

I think we should memoize this and be flexible.

Plus, this will all go away in v2.0 as ReactOnRails will be within the bundle_js_code.